src/lexers/typescript-lexer.js

```diff
@@ -0,0 +1,36 @@
+import JsxLexer from './jsx-lexer'
+import { readFileSync } from 'fs'
+import { join } from 'path'
+
+let loadedTs = null
+function loadTypeScript() {
+  if (loadedTs) {
+    return loadedTs;
+  }
+
+  try {
+    loadedTs = require('typescript')
+  } catch (e) {
+    throw new ParsingError(`You must install typescript to parse TypeScript files. Try running "yarn install typescript"`)
```

Comment:
I think you meant `yarn add`  no?

Reply:
Ah yep. I'm more of an npm user, had yarn here because I noticed that's what you use for this project 😄